integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/TestEndpoint.java

```diff
@@ -221,6 +224,154 @@ public String testModel() {
+
+        int updateByNamedParameter = Person.update("from Person2 p set p.name = 'stefNEW' where p.name = :pName",
+                Parameters.with("pName", "stefp2").map());
+        Assertions.assertEquals(1, updateByNamedParameter, "More than one Person updated");
+
+        Assertions.assertEquals(2, Person.deleteAll());
+
+        makeSavedPerson("p1");
+        makeSavedPerson("p2");
+
+        updateByIndexParameter = Person.update("from Person2 p set p.name = 'stefNEW' where p.name = ?1", "stefp1");
+        Assertions.assertEquals(1, updateByIndexParameter, "More than one Person updated");
+
+        updateByNamedParameter = Person.update("from Person2 p set p.name = 'stefNEW' where p.name = :pName",
+                Parameters.with("pName", "stefp2").map());
+        Assertions.assertEquals(1, updateByNamedParameter, "More than one Person updated");
+
+        Assertions.assertEquals(2, Person.deleteAll());
+
+        makeSavedPerson("p1");
+        makeSavedPerson("p2");
+
+        updateByIndexParameter = Person.update("set name = 'stefNEW' where name = ?1", "stefp1");
+        Assertions.assertEquals(1, updateByIndexParameter, "More than one Person updated");
+
+        updateByNamedParameter = Person.update("set name = 'stefNEW' where name = :pName",
+                Parameters.with("pName", "stefp2").map());
+        Assertions.assertEquals(1, updateByNamedParameter, "More than one Person updated");
+
+        Assertions.assertEquals(2, Person.deleteAll());
+
+        makeSavedPerson("p1");
+        makeSavedPerson("p2");
+
+        updateByIndexParameter = Person.update("name = 'stefNEW' where name = ?1", "stefp1");
+        Assertions.assertEquals(1, updateByIndexParameter, "More than one Person updated");
+
+        updateByNamedParameter = Person.update("name = 'stefNEW' where name = :pName",
+                Parameters.with("pName", "stefp2").map());
+        Assertions.assertEquals(1, updateByNamedParameter, "More than one Person updated");
+
+        Assertions.assertEquals(2, Person.deleteAll());
+
+        makeSavedPerson("p1");
+        makeSavedPerson("p2");
+
+        updateByIndexParameter = Person.update("name = 'stefNEW' where name = ?1", "stefp1");
+        Assertions.assertEquals(1, updateByIndexParameter, "More than one Person updated");
+
+        updateByNamedParameter = Person.update("name = 'stefNEW' where name = :pName",
+                Parameters.with("pName", "stefp2"));
+        Assertions.assertEquals(1, updateByNamedParameter, "More than one Person updated");
+
+        Assertions.assertEquals(2, Person.deleteAll());
+
+        try {
+            Person.update(null);
+            Assertions.fail("PanacheQueryException should have thrown");
+        } catch (PanacheQueryException x) {
+        }
+
+        try {
+            Person.update(" ");
+            Assertions.fail("PanacheQueryException should have thrown");
+        } catch (PanacheQueryException x) {
+        }
```

Comment:
You can use somthing like 
```
asssertThrows(PanacheQueryException.class, () -> Person.update(null));
```

This is more concise and more clear.

Reply:
@loicmathieu Is it a good pratice refactor all other "try and catchs" on the same file?

Reply:
According to be: YES !
Other may have different opinions :)